apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -119,16 +119,16 @@ call_contract(_Cfg) ->
     %% Now call check that we can call it.
     Fee           = 107,
     GasPrice      = 2,
-    Arg           = 42,
-    {0, CallData} = aer_data:to_binary({<<"main">>, Arg}),
+    Arg           = <<"42">>,
```

Comment:
Do we pass the arguments as strings? If so, is 42 then to be interpreted hex or decimal? In the test it's decimal.

Reply:
From the "outside" http we pass arguemnts as strings of Ring data.